src/v/raft/tests/raft_fixture.h

```diff
@@ -543,16 +536,10 @@ class raft_fixture
     bool _enable_longest_log_detection = true;
 };
 
-template<class CRTPImpl, class... STM>
```

Comment:
:D easy on the eyes.

Reply:
Really waiting for C++23 to express compile-time inheritance as easily as runtime